Spatial.tscn

```diff
@@ -182,12 +183,33 @@ __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="ImportPackDialog" type="FileDialog" parent="Control/Dialogs"]
+[node name="ImportPackDialogs" type="Control" parent="Control/Dialogs"]
+margin_right = 40.0
+margin_bottom = 40.0
```

Comment:
This is the control node that holds the other import dialogs. It probably does not need a margin. Make sure it is in the topleft corner as it will offset everything inside of it if it is not.

Reply:
Looks like this is just the default. The parent node `Dialogs` is identical in function and has the same margins. I am able to remove them though